src/components/Graph.tsx

```diff
@@ -28,6 +28,14 @@ interface Props {
 }
 
 class Graph extends React.Component <Props, States> {
+    r1_Points = [100, 100, 700, 100];
+    r2_Points = [100, 200, 700, 200];
+    r3_Points = [100, 300, 700, 300];
+    replicas = [this.r1_Points,this.r2_Points,this.r3_Points]
```

Comment:
The numbers `100` for the start of the line and `700` for the end of the line are the same for every replica. So I would create two constants for those. For the y-positions you could either have a constant for the distance between lines or an array of y-positions.